config.wyam

```diff
@@ -226,6 +226,13 @@ Pipelines.Add("Post Search Data",
     WriteFiles((doc, ctx) => "data/search/posts.json")
 );
 
+Pipelines.Add("OPML feed export", 
+    ReadFiles("data/blogs/*.yml"),

```

Comment:
You probably don't need to read the files and process them again since they alreayd get read and are available in the previous "Blog" pipeline. You should be able to replace the `ReadFiles`, `Yaml`, and `Content` modules here with a single `Documents("Blogs")`:

```
Pipelines.Add("OPML",
    Documents("Blogs"),
    Execute((inputs, ctx, _) => GetOpmlData(inputs, ctx)),
);
```